options.go

```diff
@@ -55,3 +55,13 @@ func defaultDialer() func(context.Context, string, string) (net.Conn, error) {
 func defaultLogger() Logger {
 	return Adapt(logrus.New().WithField("app", "spiceProxy"))
 }
+
+// WithConnectionCloseHandler is called when the main channel of a SPICE session is closed.
+// The "destination" parameter contains the compute node address returned by "resolveComputeAddress".
+// WithConnectionCloseHandler can be used to clean up after a SPICE connection was closed
+func WithConnectionCloseHandler(closeCallback func(destination string)) Option {
```

Comment:
Please use a function signature that allows for returning an error. Something like:
` func(destination string) error)`

Reply:
done